schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+   id INT PRIMARY KEY     NOT NULL,
+   name           CHAR (50),
+   date_of_birth  DATE,
+   escape_attempts INT,
+   neutered        BIT,
```

Comment:
- [OPTIONAL] I think it would be better to use `BOOLEAN` rather than `BIT` To follow the requirements 👌

Reply:
Thank you @Bondok6 I will fix that before merging 👍🏾